src/CodeGenerator/Assembler/FluentAdderAssembler.php

```diff
@@ -4,125 +4,86 @@
-                            '$this->%1$s[] = $%1$s;%2$sreturn $this;',
-                            $property->getName(),
-                            $class::LINE_FEED
-                        ),
-                        'docblock' => DocBlockGenerator::fromArray(
-                            [
-                                'tags' => [
-                                    [
-                                        'name' => 'param',
-                                        'description' => sprintf('%s $%s', $property->getType(), $property->getName()),
-                                    ],
-                                    [
-                                        'name' => 'return',
-                                        'description' => '$this',
-                                    ],
-                                ],
-                            ]
-                        ),
-                    ]
-                )
-            );
+        $nameParameter = ParameterGenerator::fromArray([
+            'name' => $property->getName() . 's',
+        ])->setVariadic(true);
 
-            // If property is available update its dockblock and default value.
-            if ($class->hasProperty($property->getName())) {
-                $class->getProperty($property->getName())->setDocBlock(DocBlockGenerator::fromArray([
-                    'tags' => [
-                        [
-                            'name' => 'var',
-                            'description' => $property->getType() . '[]',
-                        ],
-                    ],
-                ]));
-                $class->getProperty($property->getName())->setDefaultValue(new PropertyValueGenerator([], ValueGenerator::TYPE_ARRAY_SHORT, ValueGenerator::OUTPUT_SINGLE_LINE));
-            }
+        $returnType = TypeGenerator::fromTypeString(
+            $class->getNamespaceName() . '\\' . $class->getName()
+        );
 
-            // If setter is available update its dockblock and return value.
-            $getterMethodName = Normalizer::generatePropertyMethod('get', $property->getName());
-            if ($class->hasMethod($getterMethodName)) {
-                $class->getMethod($getterMethodName)->setReturnType('array');
-                $class->getMethod($getterMethodName)->setDocBlock(DocBlockGenerator::fromArray([
-                    'tags' => [
-                        [
-                            'name' => 'return',
-                            'description' => $property->getType() . '[]',
-                        ],
-                    ],
-                ]));
-            }
-        } catch (\Exception $e) {
-            throw AssemblerException::fromException($e);
-        }
-    }
+        $methodBody = sprintf(
+            'foreach ($%ss as $%s) {$this->%s[] = $%s;}return $this;',
+            $property->getName(),
+            $property->getName(),
+            $property->getName(),
+            $property->getName()
+        );
```

Comment:
Simpler:
        $methodBody = sprintf(
            'foreach ($%1$ss as $%1$s) {$this->%1$s[] = $%1$s;}return $this;',
            $property->getName()
        );

Reply:
Fixed by using `array_merge()`, but very nice tips, I didn't know, thanks!